config.py

```diff
@@ -49,13 +49,32 @@ class BaseConfig(object):
     # mail accounts
     MAIL_DEFAULT_SENDER = os.getenv("MAIL_DEFAULT_SENDER")
 
+
     @staticmethod
     def build_db_uri(
         db_type_arg=os.getenv("DB_TYPE"),
         db_user_arg=os.getenv("DB_USERNAME"),
         db_password_arg=os.getenv("DB_PASSWORD"),
         db_endpoint_arg=os.getenv("DB_ENDPOINT"),
-        db_name_arg=os.getenv("DB_NAME"),
+        db_name_arg=os.getenv("DB_NAME")
+    ):
+        """Build remote database uri using specific environment variables."""
+
+        return "{db_type}://{db_user}:{db_password}@{db_endpoint}/{db_name}".format(
```

Comment:
f string
Please keep the formatting consistent, and use only f-strings

Reply:
Yep, done 👍